google-cloud-storage/src/main/java/com/google/cloud/storage/StorageImpl.java

```diff
@@ -1279,6 +1281,40 @@ public Boolean call() {
     }
   }
 
+  @Override
+  public Map<BucketInfo.LifecycleRule, Boolean> deleteLifecycleRules(
+      String bucket, BucketInfo.LifecycleRule... rules) {
```

Comment:
Please consider renaming rules -> rulesToDelete

Reply:
`rules `is much better instead of `rulesToDelete`.

Reply:
I'm suggesting because you introduce lifecycleRules variable in the code. One should keep in mind, that `lifecycleRules` - stands for actual rules, when `rules` - stands for rules to be deleted.

But it's up to you.